lib/js/src/rpc/messages/Show.js

```diff
@@ -0,0 +1,316 @@
+        return this.getParameter(Show.KEY_MAIN_FIELD_1);
+    }
+
+    /**
+    * @param {String} mainField2
+    * @return {Show}
+    */
+    setMainField2(mainField2) {
+        this.setParameter(Show.KEY_MAIN_FIELD_2, mainField2);
+        return this;
+    }
+
+    /**
+    * @return {String}
+    */
+    getMainField2() {
+        return this.getParameter(Show.KEY_MAIN_FIELD_2);
+    }
+
+    /**
+    * @param {String} mainField3
+    * @return {Show}
+    */
+    setMainField3(mainField3) {
+        this.setParameter(Show.KEY_MAIN_FIELD_3, mainField3);
+        return this;
+    }
+
+    /**
+    * @return {String}
+    */
+    getMainField3() {
+        return this.getParameter(Show.KEY_MAIN_FIELD_3);
+    }
+
+    /**
+    * @param {String} mainField4
+    * @return {Show}
+    */
+    setMainField4(mainField4) {
+        this.setParameter(Show.KEY_MAIN_FIELD_4, mainField4);
+        return this;
+    }
+
+    /**
+    * @return {String}
+    */
+    getMainField4() {
+        return this.getParameter(Show.KEY_MAIN_FIELD_4);
+    }
+
+    /**
+    * @param {TextAlignment} menuParams
+    * @return {Show}
+    */
+    setAlignment(alignment) {
+        this.validateType(TextAlignment, alignment);
+
+        this.setParameter(Show.KEY_ALIGNMENT, alignment);
+        return this;
+    }
+
+    /**
+    * @return {TextAlignment}
+    */
+    getAlignment() {
+        return this.getObject(TextAlignment, Show.KEY_ALIGNMENT);
+    }
+
+    /**
+    * @param {String} statusBar
+    * @return {Show}
+    */
+    setStatusBar(statusBar) {
+        this.setParameter(Show.KEY_STATUS_BAR, statusBar);
+        return this;
+    }
+
+    /**
+    * @return {String}
+    */
+    getStatusBar() {
+        return this.getParameter(Show.KEY_STATUS_BAR);
+    }
+
+    /**
+    * @param {String} mediaClock
+    * @return {Show}
+    */
+    setMediaClock(mediaClock) {
+        this.setParameter(Show.KEY_MEDIA_CLOCK, mediaClock);
+        return this;
+    }
+
+    /**
+    * @return {String}
+    */
+    getMediaClock() {
+        return this.getParameter(Show.KEY_MEDIA_CLOCK);
+    }
+
+    /**
+    * @param {String} mediaTrack
+    * @return {Show}
+    */
+    setMediaTrack(mediaTrack) {
+        this.setParameter(Show.KEY_MEDIA_TRACK, mediaTrack);
+        return this;
+    }
+
+    /**
+    * @return {String}
+    */
+    getMediaTrack() {
+        return this.getParameter(Show.KEY_MEDIA_TRACK);
+    }
+
+    /**
+    * @param {Image} graphic
+    * @return {Show}
+    */
+    setGraphic(graphic) {
+        this.validateType(TextAlignment, graphic);
+
+        this.setParameter(Show.KEY_GRAPHIC, graphic);
+        return this;
+    }
+
+    /**
+    * @return {Image}
+    */
+    getGraphic() {
+        return this.getObject(Image, Show.KEY_GRAPHIC);
+    }
+
+    /**
+    * @param {Image} secondaryGraphic
+    * @return {Show}
+    */
+    setSecondaryGraphic(secondaryGraphic) {
+        this.validateType(TextAlignment, secondaryGraphic);
```

Comment:
Incorrect class? Probably intended:
```suggestion
        this.validateType(Image, secondaryGraphic);
```